charts/sonarqube/README.md

```diff
@@ -53,6 +61,22 @@ $ helm delete kindly-newt
 4. Browse to http://yourSonarQubeServerURL/setup and follow the setup instructions
 5. Reanalyze your projects to get fresh data
 
+### Upgrade from sonarqube-lts to this chart
+
+To install SonarQube 9.9 LTS, use this [sonarqube](https://artifacthub.io/packages/helm/sonarqube/sonarqube) Helm chart. The [sonarqube-lts](https://artifacthub.io/packages/helm/sonarqube/sonarqube-lts) Helm chart is no longer maintained and can not be used to install the new LTS.
+
+For SonarQube 9.9 LTS, the Helm chart version to use is `8.x.x`. 
+
+Before attempting any upgrade, please carefully read the general [upgrade documentation above](#upgrade), the [install-lts guide](#installing-the-lts-chart) and **backup** your db instance. Please also verify that your `values.yaml` file contains only parameters that are still compatible with the targeted chart, adjusting them if needed.
+
+As SonarQube only requires to persist the database, the general upgrade process will consist of uninstalling your instance, before installing the new LTS.
+
+If you are using an external database, you don't have any persistent data inside kubernetes. Therefore, there is no action required. 
+
+Instead, if you rely on the embedded postgres chart (**not recommended**), uninstalling the chart will keep the PVC alive. This can then be reused either:
+- by specifying `postgresql.existingClaim` in the `values.yaml` file
+- by not changing parameter values, but making sure you install the new chart in the same namespace (auto-generated name will be the same).
```

Comment:
In fact, I realize that we also plan to make it part of SonarQube documentation.
Should we maintain it in both places?

Reply:
Agree. I think it makes sense to have a single source of truth.
On Docker Hub, we refer upgrade instructions from the documentation. I feel we could do the same here.